go/ql/test/library-tests/semmle/go/dataflow/ThreatModels/threat-models-flowtest2.expected

```diff
@@ -1,7 +1,13 @@
+models
+| 1 | Source: github.com/nonexistent/sources; ; false; ExecuteQuery; ; ; ReturnValue; database; manual |
+| 1 | Summary: archive/tar; ; false; NewReader; ; ; Argument[0]; ReturnValue; taint; manual |
```

Comment:
number 1 repeated twice 😕

Reply:
The repeats are caused by a bug in the post-processing procedure. We will accept them for now and aim to fix the bug soon.